### PR TITLE
Replace FAIR.png with svg version. Fixes #23, #12.

### LINKED ISSUE #23: Figure on page 6: red lines
From FAIR, at "I", some things are undercurled red, adjust if possible

### PR DESCRIPTION
**Pull request by rcschrg**
Replaced FAIR.png with svg version.

### CODE REVIEW

**Review by JuliaHeiken — CHANGES_REQUESTED** [review]
The links are not clickable in the course. Please either remove them or embed the image in another way so that the links are working.

**Comment by rcschrg** [comment]
Changed the embedding to object for interactive svg. Note that the svg link has to changed to the main branch after approval, otherwise there would be the need for another pull request, as this branch will eventually be deleted and the cached svg will be gone after seven days.

**Review by JuliaHeiken — CHANGES_REQUESTED** [review]
Links should open in a new tab and not inside the course

**Review by JuliaHeiken — APPROVED** [review]

**Pull request merged**
